plugins/modules/synchronize.py

```diff
@@ -449,11 +456,15 @@ def main():
     ssh_connection_multiplexing = module.params['ssh_connection_multiplexing']
     verify_host = module.params['verify_host']
     link_dest = module.params['link_dest']
+    delay_updates = module.params['delay_updates']
 
     if '/' not in rsync:
         rsync = module.get_bin_path(rsync, required=True)
 
-    cmd = [rsync, '--delay-updates', '-F']
+    cmd = [rsync]
+    if delay_updates:
```

Comment:
I think it could be more readable to move this conditional block after the next one (`if rsync_password:`), so module's options would be logically ordered this way:
1. sshpass and error handling
2. rsync options

Reply:
Thanks for reviewing!
Fixed in https://github.com/ansible-collections/ansible.posix/pull/167/commits/9313bf3e68b25fe7ad177258ede6208fee9021e8.